var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -211,6 +212,10 @@ class Dealii(CMakePackage, CudaPackage):
                       'onwards. Please explicitly disable this variant '
                       'via ~{0}'.format(p))
 
+    conflicts('+nanoflann', when='@9.2.1:',
```

Comment:
this isn't quite right - we removed nanoflann after (and including) what should become the 9.3.0 release. If there is a 9.2.1 then it will have nanoflann.

Reply:
I concur with @drwells. I think that we should be safe with `when='@9.3.0:`